code/game/jobs/job/medical.dm

```diff
@@ -76,7 +76,7 @@ Your second loyalty is to your career with Moebius corp, and to your coworkers i
 
 	access = list(
 		access_moebius, access_medical_equip, access_morgue, access_surgery, access_chemistry, access_virology,
-		access_genetics
+		access_genetics, access_maint_tunnels
```

Comment:
although i would love medical and moebius to have maint. this was discussed in the past. and so far. agreed to not be a thing